Tails specific: no need to stop properly on shutdown, we are amnesic.

--- chroot.orig/etc/init.d/saned	2012-09-24 10:05:13.297055555 +0200
+++ chroot/etc/init.d/saned	2012-09-24 10:47:23.717869294 +0200
@@ -10,1 +10,1 @@
-# Default-Stop:      0 1 6
+# Default-Stop:
